pandas/tests/io/sas/test_sas7bdat.py

```diff
@@ -136,6 +136,23 @@ def test_encoding_options(datapath):
         assert x == y.decode()
 
 
+def test_encoding_infer(datapath):
+    from pandas.io.sas.sas7bdat import SAS7BDATReader
```

Comment:
This can be removed

Reply:
The whole test? Do you think it's covered by the other tests?

Reply:
This import for typing purposes. We don't using annotations in tests.

Reply:
Of course, sorry, missed the change above